opentelemetry-contrib/src/trace/id_generator/aws.rs

```diff
@@ -0,0 +1,94 @@
+use opentelemetry::trace::{SpanId, TraceId};
+use opentelemetry_sdk::trace::{IdGenerator, RandomIdGenerator};
+use std::time::{Duration, UNIX_EPOCH};
+
+/// Generates AWS X-Ray compliant Trace and Span ids.
```

Comment:
I suppose this part is copy paste?

Reply:
yeah copied as is from the opentelemetry core repo.

Reply:
some outdated references here (imports), i'll fix it.